bitrise.yml

```diff
@@ -7,12 +7,46 @@ trigger_map:
     workflow: app-store-build
   - pull_request_target_branch: develop
     workflow: qa-build
+  - pull_request_target_branch: develop
```

Comment:
This splits up the workflow so that QA Build is no longer tied to our tests. This should make bitrise builds move faster by separating the flows.